x/ecocredit/server/basket/features/msg_update_basket_fee.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Msg/MsgUpdateBasketFee
+
+  The basket fee can be updated:
+  - when the authority is a governance account address
+  - when the provided basket fee is valid
+  - the basket fee is updated
+
+  Rule: The authority is a governance account
+
+    Scenario: the authority is a governance account
+      When alice attempts to update basket fee with properties
+      """
+      {
+        "authority": "regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68",
+        "basket_fee": [
+            {
+                "denom": "uregen",
+                "amount": "1000"
+            }
+        ]
+      }
+      """
+      Then expect no error
+
+    Scenario: the authority is not a governance account
+      When alice attempts to update basket fee with properties
+      """
+      {
+        "authority": "regen1fua8speyxgempgy06gpfs0p4z32zznkqakm57s",
+        "basket_fee": [
+            {
+                "denom": "uregen",
+                "amount": "1000"
+            }
+        ]
+      }
+      """
+      Then expect error contains "expected gov account as only signer for proposal message"
+
+  Rule: The basket fee is valid
+    Scenario: the basket fee is valid
+      When alice attempts to update basket fee with properties
+      """
+      {
+        "authority":"regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68",
+        "basket_fee":[
+            {
+                "denom":"uregen",
+                "amount":"1000"
+            }
+        ]
+      }
+      """
+      Then expect no error
```

Comment:
we should also be checking the state of the param with a successful state transition here. and maybe add another case with multiple tokens in the slice